.travis.yml

```diff
@@ -1,4 +1,5 @@
-sudo: false
+dist: trusty
```

Comment:
Unfortunately, there's no support for Xenial. Trusty is the most recent supported version.